src/views/diagnosis/DiagnosisResults.js

```diff
@@ -18,13 +19,64 @@ const DiagnosisResults = ({ route }) => {
     })
     .sort((a, b) => parseFloat(b.probability) - parseFloat(a.probability));
 
+  const [submitDiagnosis, setDiagnosis] = useState([]);
```

Comment:
To jest tylko lokalnie i nie będziemy potrzebować kontekstu?

Reply:
Na razie jest lokalnie, później będzie w kontekście